unbricking.md

```diff
@@ -0,0 +1,193 @@
+# G-010S-A Unbricking
+
+If anything goes wrong during an update the SFP module can be unbricked through
+a serial console.
+
+## Preparation / Unlock Bootloader
+<a name="preparation"></a>
+
+By default the bootloader is locked and will not allow interrupting the boot.
+Unlock the bootloader while the SFP module is still working using:
+```
+fw_setenv asc0 0            # Fully activate console
+fw_setenv bootdelay 5       # Prompt for access to u-boot console
+fw_setenv preboot           # Avoid console hanging while initializing network
+```
+
+## Serial Console Cable
+
+The serial console can be accessed with a UART to SFP cable. One tested UART
+board is Diligent PmodUSBUART. Ideally the UART board provides 3.3V for the SFP
+module. The SFP connector (e.g., Molex 74441-0010) is connected to the UART on
+4 pins:
+| UART Pin     | SFP Pin  |
```

Comment:
Since there is often confusion with TX/RX being wrongly connected, I would use `UART Board/Cable` or something similar, to make it obvious we are not talking about the G-010S-A pinout.

Reply:
done